Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -965,10 +1013,10 @@ QWidget* EditorActionsHandler::CreateExpander()
     return expander;
 }
 
-QWidget* EditorActionsHandler::CreateLabel(const AZStd::string& text)
+QWidget* EditorActionsHandler::CreatePlayControlsLabel()
 {
     QLabel* label = new QLabel(m_mainWindow);
-    label->setText(text.c_str());
+    label->setText("Play Controls");
     return static_cast<QWidget*>(label);
```

Comment:
Do we need this `static_cast<QWidget*>` here? I would think the `QLabel` would just get downcast to the `QWidget` implicitly, but maybe not?